### ISSUE TITLE
Mechanism for storing original concept IDs on import

### ISSUE THREAD

**Issue by rgarner** [rgarner/semantically-taggable#6]
Currently we store the SKOS `prefLabel`.  Good practice dictates this is guaranteed unique within a scheme.  There is provision (in the form of an `original_id` column) to hold the original concept ID during the import.  However, there would need to be a mechanism to bring it in.  Currently, all the taggings are done AATO-style, e.g.

```
   article.dg_topic_list = ['Health', 'Travel']
```

which would need to become something like

```
   article.dg_topic_id_list = { 1 => 'Health', 2 => 'Travel' }
```

**Comment by rgarner** [comment]
It's likely that we'll need this for permanent URLs anyway, so will add support.  A `prefLabel` for a concept might change, but its ID should not, which means we should always be able to have a URL like `/doc/concepts/dg_topics/2260` instead of `/tags/dg_concepts/directgov%20taxonomy`

**Comment by rgarner** [comment]
SKOS Import.  Closed by 182741f57f83444917b35a0c28c2b6050acad620.
